src/testFixtures/java/org/opensearch/knn/KNNRestTestCase.java

```diff
@@ -1002,6 +1027,37 @@ protected Request constructScriptedMetricAggregationSearchRequest(
         return request;
     }
 
+    protected Request constructScriptFieldsContextSearchRequest(
+        String indexName,
```

Comment:
```suggestion
        final String indexName,
```

Reply:
ACk

Reply:
final?

Reply:
Can we make all the params final which are not changed in the below method,

Reply:
Ack